### ISSUE TITLE
Errors in RTL 

### ISSUE THREAD

**Issue by AI-IMED** [yanus171/Handy-News-Reader#479]
• [Only, in Articles' list], when the title of Arabic articles or any RTL language starts with text from LTR, this will result in losing the correct RTL format for the title, this error is very annoying when I need a quick preview of the article titles only

![Screenshot_20200423_114019](https://user-images.githubusercontent.com/38974362/80087787-42133400-8564-11ea-9217-c8d5d5969562.png)

Alexey, Please, save me from this problem.

You can try these feeds:
https://www.electrony.net/feed/
http://www.aitnews.com/feed

• When the action bar is hidden and the article or any other article is reopened, this will cause the RTL alignment to be lost in the list

![Screenshot_20200423_111351](https://user-images.githubusercontent.com/38974362/80088461-38d69700-8565-11ea-8c67-f21567ac9779.png)

Thanks in advance

**Comment by afmbsr** [comment]
Thank you very much, the following problem has been fixed:

> • [Only, in Articles' list], when the title of Arabic articles or any RTL language starts with text from LTR, this will result in losing the correct RTL format for the title, this error is very annoying when I need a quick preview of the article titles only

But regarding the next issue, unfortunately, nothing has changed:

> • When the action bar is hidden and the article or any other article is reopened, this will cause the RTL alignment to be lost in the list

I can clarify that more, the problem begins when returning to Articles' list, specifically when opening any article again, and continues with the program even when it is restarted, as long as the action bar is hidden

**Comment by yanus171** [comment]
You speak about article list, but your screenshot is about menu in article 
text. You'd better to send me a video

**Comment by afmbsr** [comment]
Sorry, I mean the list in the article

**Comment by afmbsr** [comment]
I created a gif

![20200425_152041](https://user-images.githubusercontent.com/38974362/80279909-167d7e80-8709-11ea-8f05-0b4726e5a939.gif)

**Comment by yanus171** [comment]
Finally I have got it. You mean context menu items.

**Comment by afmbsr** [comment]
exactly

**Comment by AI-IMED** [comment]
But the alignment inversion of context menu items was not fixed when articles were opened in full screen

**Comment by yanus171** [comment]
Notice that only untranslated texts have bugs

**Comment by AI-IMED** [comment]
> Notice that only untranslated texts have bugs

I don't think so, you can notice the shot on the left in this old photo that all the texts are translated into Arabic.
https://user-images.githubusercontent.com/38974362/80088461-38d69700-8565-11ea-8c67-f21567ac9779.png

**Comment by yanus171** [comment]
Why do you consider an old photo from an old app version?

**Comment by AI-IMED** [comment]
Hi @yanus171 
I have finished improving and completing the Arabic translation.

**Comment by yanus171** [comment]
Try https://github.com/yanus171/Handy-News-Reader/releases/tag/v0.19.15

**Comment by AI-IMED** [comment]
Arabic translation is not merged

**Comment by yanus171** [comment]
Try https://github.com/yanus171/Handy-News-Reader/issues/683#issuecomment-1875926725

**Comment by AI-IMED** [comment]
https://github.com/yanus171/Handy-News-Reader/assets/38974362/167a8448-2250-4e69-8516-2f48d1c0faed

As I expected, all the elements have been translated and the error still exists

**Comment by yanus171** [comment]
But the menu item text are all aligned at the same side now. What's wrong else?

**Comment by AI-IMED** [comment]
https://github.com/yanus171/Handy-News-Reader/assets/38974362/28bacafb-c6ce-488b-a2b4-c996761e1e3d

It is supposed to be such a direction and alignment

**Comment by AI-IMED** [comment]
![٢٠٢٤٠١٠٤-٠٩٤٣٠٦](https://github.com/yanus171/Handy-News-Reader/assets/38974362/78d37414-c0c5-473b-b6eb-c23ca6eba21e)
I also hope you find a solution to the problem of overlapping the Select All as Read button with the Undo option

**Comment by yanus171** [comment]
Is this video from old app version? What version?

**Comment by AI-IMED** [comment]
The Video from the latest beta version 0.19.16
